tests/testthat/test-call_derivation.R

```diff
@@ -90,3 +90,94 @@ test_that("call_derivation works", {
 
   expect_dfs_equal(expected_output, actual_output, keys = c("USUBJID", "AESEQ"))
 })
+
+
+test_that("Error is thrown if ... has no arguments", {
```

Comment:
Update `test_that("Error is thrown if ... has no arguments"` to test_that("call_derivation Error is thrown if ... has no arguments".
Function name should be added for each of the new tests added, as existing tests have function name in the test_thsat call.